tools/topology/sof-tgl-max98373-rt5682.m4

```diff
@@ -186,14 +197,6 @@ dnl SSP_CONFIG_DATA(type, idx, valid bits, mclk_id)
 dnl mclk_id is optional
 dnl ssp1-maxmspk, ssp0-RTHeadset
 
-#SSP 1 (ID: 7)
-DAI_CONFIG(SSP, 1, 7, SSP1-Codec,
-	SSP_CONFIG(DSP_B, SSP_CLOCK(mclk, 38400000, codec_mclk_in),
-		      SSP_CLOCK(bclk, 9600000, codec_slave),
-		      SSP_CLOCK(fsync, 48000, codec_slave),
-		      SSP_TDM(8, 25, 3, 3),
-		      SSP_CONFIG_DATA(SSP, 1, 24)))
-
```

Comment:
likewise SSP1 is used for the amp but the config removed here?

Reply:
yes, it will be configured inside the common smart amp m4: sof-smart-amplifier.m4